mailbox/memory/src/test/java/org/apache/james/mailbox/inmemory/InMemoryMessageManagerTest.java

```diff
@@ -18,22 +18,55 @@
  ****************************************************************/
 package org.apache.james.mailbox.inmemory;
 
+import static org.apache.james.mailbox.fixture.MailboxFixture.ALICE;
+import static org.apache.james.mailbox.fixture.MailboxFixture.BOB;
+import static org.apache.james.mailbox.fixture.MailboxFixture.INBOX_ALICE;
+
+import org.apache.james.mailbox.MailboxManager;
+import org.apache.james.mailbox.MailboxSession;
+import org.apache.james.mailbox.MailboxSessionUtil;
+import org.apache.james.mailbox.fixture.MailboxFixture;
 import org.apache.james.mailbox.inmemory.manager.InMemoryIntegrationResources;
-import org.apache.james.mailbox.store.AbstractMessageManagerTest;
+import org.apache.james.mailbox.store.MessageManagerContract;
 import org.apache.james.mailbox.store.MessageManagerTestSystem;
-import org.junit.Before;
+import org.junit.jupiter.api.BeforeEach;
 
-public class InMemoryMessageManagerTest extends AbstractMessageManagerTest {
+class InMemoryMessageManagerTest implements MessageManagerContract {
 
-    @Override
-    @Before
-    public void setUp() throws Exception {
-        super.setUp();
+    MailboxManager mailboxManager;
+    MailboxSession aliceSession;
+    MailboxSession bobSession;
+
+    @BeforeEach
+    void setUp() throws Exception {
+        aliceSession = MailboxSessionUtil.create(ALICE);
+        bobSession = MailboxSessionUtil.create(BOB);
+        MessageManagerTestSystem testSystem = createTestSystem();
+        mailboxManager = testSystem.getMailboxManager();
+
+        testSystem.createMailbox(INBOX_ALICE, aliceSession);
+        testSystem.createMailbox(MailboxFixture.OUTBOX_ALICE, aliceSession);
+        testSystem.createMailbox(MailboxFixture.SENT_ALICE, aliceSession);
+        testSystem.createMailbox(MailboxFixture.INBOX_BOB, bobSession);
```

Comment:
Such a setUp content have nothing to do in the implementation

Reply:
I think I had a test isolation issue when I tried to do like this, but will dig more in that today